prac_09/taxi_simulator.py

```diff
@@ -0,0 +1,84 @@
+from prac_06.car import Car
```

Comment:
You don't need to import the car class as it is already the parent class for the other classes you import.